### PR TITLE
Tim/docker travis

### PR DESCRIPTION
**Pull request by readevalprint**
This is good stuff

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -11,6 +12,7 @@ core:
    - ./gather2-core:/opt/gather2-core/
   net: "host"
   command:  /opt/gather2-core-env/bin/python /opt/gather2-core/manage.py runserver 0.0.0.0:8000
+  privileged: true
```
Comment (peritus) [inline docker-compose.yml]: You're privileged!

File: gather2-core/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,4 @@
 Django>=1.9a1
-jsonschema
```
Comment (peritus) [inline gather2-core/requirements.txt]: BBBBbbuutt, we need it!

File: .travis.yml [hunk 3]
```diff
@@ -1,24 +1,30 @@
...
+  - pip install flake8 docker-compose
 
 before_script:
-  - psql -c 'create database travis_ci_test;' -U postgres
+  - docker-compose run init
 
 script:
-  - invoke check
-  - invoke test -c Travis
+  - flake8 --exclude migrations --ignore E501,C901 ./
```
Comment (peritus) [inline .travis.yml]: Duplicate code with tasks.py

File: gather2-core/requirements.txt (hunk 2 above)
Comment (readevalprint) [inline gather2-core/requirements.txt]: was already there?!

**Comment by readevalprint** [comment]
@peritus check it out por favor

**Pull request merged**